services/sensorhub-service-video/src/main/java/org/sensorhub/impl/service/sos/video/VideoSerializerDescriptor.java

```diff
@@ -8,55 +8,52 @@
 WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License
```

Comment:
I think you moved the STAServiceDescriptor file instead of copying it. Need to restore the original file for STA service

Reply:
restore the service: e052a5de4916edac31fdddedf97b85c73536ace1